tilauskasittely/tilaus_osto.php

```diff
@@ -1950,3 +1979,32 @@ function verify(){
 }
 
 require "inc/footer.inc";
+
+/**
+ * @param array $params
+ *
+ * @return array Ensimm�inen ja viimeinen tilausrivi annetulla tuotenumerolla kyseisess� tilauksella
+ */
+function hae_eka_ja_vika_tilausrivi($params) {
+  global $kukarow;
+
+  $tilausnumero = $params['tilausnumero'];
+  $tuoteno      = $params['tuoteno'];
+
+  $query = "SELECT tunnus
+            FROM tilausrivi
+            WHERE yhtio = '{$kukarow["yhtio"]}'
+            AND otunnus = '{$tilausnumero}'
+            AND tuoteno = '{$tuoteno}'
+            ORDER BY tunnus";
```

Comment:
@tiere Tämä huomioi nyt myös poistetut, tyyppi=D, rivit. Lisää ehto and tyyppi='O'.